### PR TITLE
Brings centcom slightly up to date

### PR DESCRIPTION
**Pull request by francinum**
Also fixes the big ERT shuttle, it's still not perfect, but it's transit actually works.
Image:

### CODE REVIEW

**Comment by Alterist** [comment]
Neat, might want to remove those mech suits just in case, considering that this area was never meant to be accessed in the first place

**Comment by Rohesie** [comment]
The ERT shuttles will have to either have their own bases or be cut off from CentComm. We have all sorts of ERTs spawning there.

File: config/admins.txt [hunk 1]
```diff
@@ -75,3 +75,4 @@ virtualjohn - Admin
 wjohnston - Admin
 wubli - Admin
 wilchen - Admin
+francinum - Admin
```
Comment (Rohesie) [inline config/admins.txt]: Care to put this in alphabetical order?

File: code/game/machinery/doors/shutters.dm [hunk 2]
```diff
@@ -86,6 +86,12 @@
 	ex_act(severity) //immune to explosions
 		return
 
+/obj/machinery/door/poddoor/shutters/almayer/open
+	density = 0
+	opacity = 0
+	unacidable = 1
```
Comment (Rohesie) [inline code/game/machinery/doors/shutters.dm]: `TRUE`, `FALSE` and all that.

**Pull request merged**